### ISSUE TITLE
Automate tilejson generation

### ISSUE THREAD

**Issue by rmarianski** [tilezen/vector-datasource#874]
The tilejson tends to [fall out of date](https://github.com/tilezen/vector-datasource/issues/872) as we make changes to layers. Ideally, we'd be able to create an automated process to keep them fresh.

**Comment by nvkelso** [comment]
Should we include more property keys? We have a ton, but only list 1 up to 6 for roads in this config file.

**Comment by rmarianski** [comment]
We can try to infer them from the yaml file, but it won't be exactly right. Some queries include properties directly, and we have transforms that update the properties too.

We had also previously discussed being able to generate some documentation and these tilejson files from a single source. Or maybe there's a way that we can annotate the documentation itself to allow us to extract the relevant data to generate the tilejson.

**Comment by rmarianski** [comment]
Thinking about this a little bit more, we should be able to generate this by annotating the yaml and python transform configuration. Additionally, it might be worthwhile to convert the documentation to a template, and have the lists of possible kind values be injected in from this generated list.

**Comment by zerebubuth** [comment]
I agree - we can be more explicit about the values we're including, or the lists of keys we expect from the database. That, plus documentation strings and "for more information" links should be able to provide us pretty comprehensive docs in both HTML / Markdown and TileJSON.

**Comment by zerebubuth** [comment]
Since #1771, this might be within reach? Worth another look soon?
